Profiler/utils/Debug.h

```diff
@@ -9,32 +9,16 @@
 class Debug
 {
 public:
+	Debug();
```

Comment:
If you agree with my comment about this being a singleton, please make constructor and destructor private